README.md

```diff
@@ -57,10 +59,12 @@ To install LINKS in a specified directory:
 ```
 * *These steps worked on a CentOS 7 system with 128 CPU Intel(R) Xeon(R) CPU E7-8867 v3 @ 2.50GHz
 
+The `make install` does NOT install `./bin/LINKS-make` and `./bin/LINKS.pl` files nor `./tools/*.pl` files. It is just incomplete. Run LINKS from the compiled source tree directory until that is fixed.
```

Comment:
The files in `bin` folder are implemented in shell or Perl language that's why they doesn't need to be compiled by `autotools`. Can you describe what error did this caused to you?

Reply:
Hi, well simply the `make install` procedure should install all necessary files, not just some. Then, I should be able to just include the `$prefix/bin` folder into my `PATH` and eventually `$prefix/lib` into `LIBDIR` and be done with that. Just include the perl files in the `install` target.

I have no idea if from the btlib only the header files are need or also some library, I did not bother to study that.

Reply:
Hi Martin, thanks for pointing this out. Support for `install` will be developed soon. Currently, we suggest users add `bin` folder to their PATH.